pvr.vuplus/addon.xml.in

```diff
@@ -71,7 +71,7 @@
     <description lang="de_DE">VU+ -Oberfläche; Unterstützt Live TV &amp; Aufnahmen, EPG und Timer.</description>
     <description lang="el_GR">Frontend για το VU+. Υποστηρίζει ροές Live TV &amp; Εγγραφές, EPG, Χρονοδιακόπτες.</description>
     <description lang="en_AU">VU+ frontend; supporting streaming of Live TV &amp; Recordings, EPG, Timers.</description>
-    <description lang="en_GB">Enigma2 frontend; supporting streaming of Live TV &amp; Recordings, EPG, Timers, Autotimers.</description>
+    <description lang="en_GB">Enigma2 PVR client addon; frontend supporting streaming of Live TV & Recordings, EPG, Timers, Autotimers. Compatible with Enigma2-based TV-boxes from different manufacturers, such as Amiko, DBox2, Dreambox, Gigablue, SAB, VuPlus/Vu+, WeTek, Zgemma, Xtrend, and more.</description>
```

Comment:
`&` -> `&amp`

Reply:
Not sure I like the new advertisement for several vendors. If those vendors will break something on their end, users will blame Kodi as we stated here that we support them. Please remove the reference to the different vendors.

Reply:
How about with that extended clarification of no guarantees as a disclaimer. Not listing any specific boxes/manufactures at all would probabably be a mistake as there are no actual boxes that are called "Enigma2" or even really advertise as "Enigma2-based" or "Enigma2 compatible" so could be a good idea to at least list some that "should be API compatible" even if no guarantees are being promised.